pkg/selinux/selinux.go

```diff
@@ -269,20 +268,37 @@ func SelinuxGetEnforceMode() int {
 	return Disabled
 }
 
+func mcsPath(mcs string) string {
+	filename := fmt.Sprintf("/run/rkt/%s", mcs)
+	return filename
+}
+
 func mcsAdd(mcs string) error {
-	if mcsList[mcs] {
-		return fmt.Errorf("MCS Label already exists")
+	filename := mcsPath(mcs)
+	file, err := os.OpenFile(filename, os.O_CREATE|os.O_EXCL|os.O_RDONLY, 0644)
```

Comment:
The directory `/run/rkt` seems to be created by [metadata_service.go:unixListener()](https://github.com/coreos/rkt/blob/master/rkt/metadata_service.go#L585) if the metadata service is used, but it might not exist otherwise.

Can you create the directory?

Actually, do all modern distributions set up a `/run` tmpfs? Or older distros like Centos 6.x (that we try to support, see #1443).